apps/frontend/src/hooks/aave/useAaveUserReservesData.tsx

```diff
@@ -40,10 +40,12 @@ export const useAaveUserReservesData = () => {
       reservesData: ReservesDataHumanized,
       signer: ethers.Signer,
     ) => {
+      // const user = await signer.getAddress()
+      const user = '0xF754D0f4de0e815b391D997Eeec5cD07E59858F0';
```

Comment:
i think the commented code is the correct one. In case we'll need it because the feat is WIP, I recomment to add a TODO to not forget in the future.

Reply:
Totally! We need until calculations in bob don't break ) Added the TODO